posts/inside-rust/2020-07-09-lang-team-path-to-membership.md

```diff
@@ -0,0 +1,117 @@
+* Participate in discussions and in particular help to create summaries or otherwise resolve technical disputes in a productive way
+
+We realize that we can identify people who are doing some of those
+things already and approach to see if they are interested in lang-team
+membership.  If so, we can look for opportunities to complete some of
+the other bullets.
+
+## A bit of background: project groups
+
+We've not been blogging a lot about the idea of project groups and the
+like so let me give just a bit of background. In short, the idea is
+that we are trying to "intercept" the RFC process earlier by
+introducing a "pre-step" called a Major Change Proposal
+(MCP). (Terminology still subject to change as we experiment here.)
+
+The idea is that if you have an idea you'd like to pursue, you can
+file an MCP issue and describe the high-level details. If the idea
+catches the eye of somebody within the team, we will create a
+**project group** to pursue the idea, with that member serving as the
+**lang team liaison** and you (or others) serving as the **group
+lead**.
+
+A **project group** doesn't have to be a huge group of people. It
+might even just be one or two people and a dedicated Zulip stream.
+The idea is that the group will work out the design space and author
+RFCs; once the RFCs are accepted, the group can also pursue the
+implementation (though the set of people involved may shift at that
+point), and hopefully see the idea all the way through to
+stabilization.
+
+### Growing the set of folks who can serve as liaison
+
+One of the things we talked about was the proper role for a project
+group liaison. As described in the previous paragraph, a liaison was
+basically a member of the team who would follow along with the design
+and help to keep the rest of the team up to date. 
+
+But we realized that if we limit liaisons to team members, then this
+is incompatible with this idea of a "path to membership" where people
+can "trial run" lang-team activities.  It's also somewhat incompatible
+with a core goal, which is that the experience of someone who is *not*
+on a team and someone who *is* on a team ought to be awfully close,
+and that we should be careful when creating distinctions.
+
+Therefore, we discussed the idea of saying that liaisons don't have to
+be team members, they just have to be people who are heavily involved
+in the project and who can be trusted to create regular updates for
+the lang-team and keep the rest of the team in the loop. 
+
+In particular, this can also be a useful stepping stone towards full
+lang-team membership -- although it doesn't have to be. It's useful to
+have people serve as liaisons even if they don't really have time or
+interest in being on the lang team.
+
+### Conclusion
+
+We concluded that we'll start experimenting with "non-team-member
+liaisons", and that people who are maybe interested in that role can
+reach out privately to Josh Triplett or myself. Further, we'll work to
```

Comment:
This post may make the rounds, and authorship information on the posts is a note at the top; you might want to give your name explicitly.